string cmd
float64 x
float64 y
float64 z
float64 roll
float64 pitch
float64 yaw
int64 id
float64 slice_radius
float64 slice_height
float64 slice_thick
